nuxeo-features/nuxeo-platform-comment/nuxeo-platform-comment/src/main/java/org/nuxeo/ecm/platform/comment/impl/CommentManagerImpl.java

```diff
@@ -525,4 +525,14 @@ public DocumentRef getTopLevelCommentAncestor(CoreSession s, DocumentRef comment
     public DocumentRef getCommentedDocumentRef(CoreSession session, DocumentModel commentDocumentModel) {
         throw new UnsupportedOperationException();
     }
+
+    @Override
+    protected boolean hasComments(CoreSession session, DocumentModel document) {
+        throw new UnsupportedOperationException();
+    }
+
+    @Override
+    protected boolean hasCommentsByUser(CoreSession session, DocumentModel document, String author) {
+        throw new UnsupportedOperationException();
+    }
```

Comment:
if we decide to not make the two methods `abstract` you will have the default impl on `abstract class`